src/lib/BulkDataClient.ts

```diff
@@ -56,6 +58,7 @@ export interface BulkDataClientEvents {
         response           : Response
         capabilityStatement: fhir4.CapabilityStatement
         requestParameters  : Record<string, any>
+        responseHeaders    : Types.ResponseHeaders | undefined
```

Comment:
Just to save a few bytes and simplify, here and in few other places below, `responseHeaders    : Types.ResponseHeaders | undefined` could be replaced with `responseHeaders   ?: Types.ResponseHeaders`

Reply:
Fixed here and other instances of `responseHeaders : Types.ResponseHeaders | undefined`